src/main/java/pl/cyber/trainees/SimpleApp.java

```diff
@@ -1,29 +1,36 @@
-package pl.cyber.trainees;
 
-import pl.cyber.trainees.dziedziczenie.Kolor;
-import pl.cyber.trainees.dziedziczenie.Model;
 
-public class SimpleApp {
+package pl.cyber.trainees;
+import pl.cyber.trainees.wyjasnienia.zadanieDomowe29tydz;
 
+public class SimpleApp {
 
     public static void main(String[] args) {
 
 
-        Model modelPojazdu1 = new Model ();
-        var modelPojazdu2 = new Model ("VW", "2020", "przod", "Passat",
-    10000, 2.0,"NIEBIESKI");
+        zadanieDomowe29tydz zadanie = new zadanieDomowe29tydz();
```

Comment:
klasa zadanieDomowe29tydz powinna zacząć się z wielkiej litery, gdyż nie jest to nazwa metody a klasy.